stable/influxdb/Chart.yaml

```diff
@@ -1,5 +1,6 @@
 name: influxdb
-version: 0.8.3
+version: 0.8.4
+appVersion: 0.8.4
```

Comment:
looking at https://github.com/AdamDang/charts/blob/5afd5234e898ab4d0a79e28016d6bceb554b8bca/stable/influxdb/values.yaml#L5, the appVersion should probably be `1.4`